gallery/visualtests/darkModeSettings.js

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+
+const { setupBrowser } = require('./testUtils');
+
+describe('Dark Mode Settings', function() {
+  const { getPage,
+    checkFullPageScreenshot,
+    setThemeOverride,
+    setThemingEnabled,
+    wait
+  } = setupBrowser('#/');
+
+  function themeOverrideTest(theme) {
+    return async function() {
+      await setThemeOverride(theme);
+      await wait(400);
+      await checkFullPageScreenshot();
+    };
+  }
+
+  beforeEach(async function() {
+    await getPage().setViewport({ width: 1366, height: 1000 });
+  });
+
+  it('sets display theme according to browser preference', themeOverrideTest(null));
+
+  it('sets display theme to dark when themeOverride is set to dark', themeOverrideTest('dark'));
+
+  it('sets display theme to light when themeOverride is set to light', themeOverrideTest('light'));
+
+  describe('when theme changes are disabled', function() {
+    beforeEach(async function() {
+      await setThemingEnabled(false);
+    });
+
+    it('sets display theme to light mode', async function() {
+      await checkFullPageScreenshot();
+    });
+
+    it('disallows browser preference', themeOverrideTest(null));
```

Comment:
Aren't these two the same?

Reply:
> Aren't these two the same?

yes actually, it hadn't crossed my mind. Which 'it' statement would you prefer to keep?

Reply:
> yes actually, it hadn't crossed my mind. Which 'it' statement would you prefer to keep?

The second one, it's most similar to the others.

Reply:
> The second one, it's most similar to the others.

Removed here: [caa023f](https://github.com/sonatype/sonatype-react-shared-components/pull/808/commits/caa023f2a4a2a08d59bdf2bb36444a12682ba8cc)